config/settings/base.py

```diff
@@ -657,4 +657,4 @@
 # This is the official and final production phase date of the employee record feature.
 # It is used as parameter to filter the eligible job applications for the feature.
 # (no job application before this date can be used for this feature)
-EMPLOYEE_RECORD_FEATURE_AVAILABILITY_DATE = timezone.datetime(2021, 9, 27)
+EMPLOYEE_RECORD_FEATURE_AVAILABILITY_DATE = timezone.datetime(2021, 7, 1)
```

Comment:
Told you so 🤣

Reply:
C'est maintenant que les emmerdes commencent. J'espérais avoir encore une vraie nuit de sommeil.
So much for the "progressive opening", la c'est 3 mois directs qui rentrent